editor/editor_feature_profile.cpp

```diff
@@ -12,7 +12,7 @@ const char *EditorFeatureProfile::feature_names[FEATURE_MAX] = {
 	TTRC("Scene Tree Editing"),
 	TTRC("Import Dock"),
 	TTRC("Node Dock"),
-	TTRC("Filesystem Dock")
+	TTRC("Filesystem/Import Dock")
```

Comment:
Does disabling the FileSystem dock (BTW, capital S) also force disabling the Import dock?
If so, it should likely be "FileSystem and Import Dock**s**".

Reply:
Done.